blueprints/ember-frost-list/index.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-frost-core', target: '>=0.0.13 <2.0.0'}
+        {name: 'ember-frost-core', target: '>=0.0.13 <2.0.0'},
+        {name: 'smoke-and-mirrors', target: '>=0.5.2 <1.0.0'}
```

Comment:
I'd lock smoke-and-mirrors to a particular minor version, we don't know if the API will remain stable until 1.0